overlay.d/07c9s/bootstrap/pre-pivot.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script is called by the OKD installer and runs in the
+# local context of the bootstrap node.
+# It rebases the booted operating system (e.g. Fedora CoreOS)
+# to CentOS Stream CoreOS 9
+
+# Load common functions
+. /usr/local/bin/release-image.sh
```

Comment:
I think it's problematic to have content shipped in the operating system read code injected from outside (in this case via Ignition).  Instead I think we should have a cleaner, well-defined API between the two.

Hmm, in this case though, this code is so small that what's the value in having it shipped by the OS image instead of fully injecting it via Ignition?

Since it's only for the bootstrap node, that seems to make even more sense since we don't care about it on production nodes.